docs/ami.md

```diff
@@ -0,0 +1,335 @@
+# Archipelago Multi-Importer
+
+[Archipelago Multi-Importer (AMI)](https://github.com/esmero/ami) is a module for batch/bulk/mass ingests of Archipelago digital objects and collections. AMI also enables you to perform batch administrative actions, such as updating, patching/revising, or deleting digital objects and collections.
+
+### *Work-in-Progress Notes*
+
+*Please be aware that the version of AMI shipped with Archipelago 1.0.0-RC1 provides much of the core batch functions useful for getting started working with large amounts of content, but is not the full or final version of this module. This documentation page will be updated with successive AMI and Archipelago releases.*
+
+## Getting started with AMI
+
+You can access AMI through the `AMI Sets` tab on the main Content page found at `/admin/content`
+
+  ![Content AMI Sets](/imgs/ami/ContentAMIsets.jpg)
+
+or directly at `/amiset/list`
+
+  ![AMI Sets List](/imgs/ami/AMIsetsList.jpg)
+
+## Google Sheets API Congifuration
+
+If you plan on using the Google Sheets Importer option, you will need to:
+  * [Configure the Google Sheets API](/docs/googleapi.md)
+
+## Spreadsheet Formatting Overview
+
+There are multiple ways a spreadsheet/CSV file can be structured to work with AMI, depending on the data transformation and mapping you will be using.
+
+<details><summary>Click to view Basic Requirements and Recommended Options</summary>
+<span>
+
+- For most standard AMI ingests, each Row of your spreadsheet/CSV will correspond to a single Digital Object or Collection.
+- Columns in your spreadsheet/CSV can be mapped to different data (files) and metadata elements (label, description, subjects, etc.).
+
+- It is recommended that different types of files are placed into separate columns--"images", "documents", "models", "videos", "audios".
+  - Filepaths can point to remote files, to existing files within your docker container, and s3 (or other storage type/location that is accessible to Archipelago).
+      - Example path for existing file within docker container:
+      `/var/www/html/d8content/myAMIimage.jpg`
+      - Example s3 path:
+      `s3://myAMIuploads/myAMIdocument.pdf`
+      - Example remote filepath:
+      `https://dogsaregreat.edu/dogs.tiff`
```

Comment:
I was kinda looking forward to see a dog here 🐶

Reply:
😸 hard to believe such a site does not (yet) exist?